root/static/scripts/edit/URLCleanup.js

```diff
@@ -1000,9 +1000,25 @@ const CLEANUPS = {
       if (m) {
         const prefix = m[1];
         if (_.includes(LINK_TYPES.videochannel, id)) {
+          if (prefix === 'video/') {
+            return {
+              error: l(
+                `Please link to a channel, not a specific video.
+                 Videos should be linked to the appropriate
+                 recordings instead.`,
```

Comment:
All of these error messages for video relationships should probably be defined in a set of common error messages once.

Reply:
They're not all exactly the same though, since some cases are slightly different than others.

Reply:
Most of error messages in this commit are used at least two times, if not three times. If these messages are expected to be reused for more video websites, constants seem worth to have.